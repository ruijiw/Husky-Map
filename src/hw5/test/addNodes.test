# Specification testing file.
# Tests that a graph can be created, multiple of nodes and
# can be added, and all added nodes should be printed out.


# Create a graph
CreateGraph graph1

# Add multiple nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6
AddNode graph1 n7

# Print the nodes in the graph
ListNodes graph1