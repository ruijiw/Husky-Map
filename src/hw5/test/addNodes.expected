# Specification testing file.
# Tests that a graph can be created, multiple of nodes and
# can be added, and all added nodes should be printed out.


# Create a graph
created graph graph1

# Add multiple nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1
added node n7 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3 n4 n5 n6 n7